src/GNNGraphs/transform.jl

```diff
@@ -412,6 +412,72 @@ function add_edges(g::GNNHeteroGraph{<:COO_T},
              ntypes, etypes)
 end
 
+"""
+    perturb_edges([rng], g::GNNGraph, perturb_ratio)
```

Comment:
You changed the docstring but you didn't change the implementation...

Reply:
fixing